driver/pace/driver/run.py

```diff
@@ -27,6 +29,8 @@
 from pace.util.grid import DampingCoefficients
 from pace.util.namelist import Namelist
 
+from .report import collect_data_and_write_to_file
```

Comment:
why are you opting for relative imports here?

Reply:
Might be other reasons, but also to some degree because I asked for it. I think either should be fine for anyone to merge, but relative is nice in that it clues you in to "this is from the same package", especially when we have a multi-package structure for our code.

Reply:
Yes either works, will keep it as relative import unless there's a strong preference for absolute.